File: haikan-parent/haikan-manager-interface/src/main/java/com/haikan/manager/service/MonitoredService.java
package com.haikan.manager.service;

import com.haikan.pojo.TbMonitored;

import entity.PageResult;
import entity.Result;

/**
 * 受监控对象服务层接口
 * @author Administrator
 *
 */

public interface MonitoredService {

	/**
	 * 添加受监控对象信息
	 * @param tbMonitored
	 */
	public void add(TbMonitored tbMonitored);

	/**
	 * 分页查询
	 */
	public PageResult findPage(int pageNum,int pageSize);
	
	/**
	 * 根据id查询受监控对象信息
	 * @param id
	 */
	public TbMonitored findOne(Long id);
	
	/**
	 * 修改受监控对象信息
	 * @param tbMonitored
	 * @return
	 */
	public void update(TbMonitored tbMonitored);
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] ids);
}

File: haikan-parent/haikan-manager-web/src/main/webapp/js/service/monitoredService.js
app.service('monitoredService',function($http){
	
	this.add=function(imageUrl){
		return $http.get('../monitored/add.do?imageUrl='+imageUrl);
	}
	
	this.update=function(entity){
		return $http.post('../monitored/update.do',entity);
	}
	
	//分页查询
	this.findPage=function(pageNum,pageSize){
		return $http.get('../monitored/findPage.do?pageNum='+pageNum+'&pageSize='+pageSize);
	}
	
	//根据id查询监控信息
	this.findOne=function(id){
		return $http.get('../monitored/findOne.do?id='+id);
	}
	
	//批量删除
	this.dele=function(ids){
		return $http.get('../monitored/delete.do?ids='+ids);
	}
});